lte/gateway/deploy/roles/magma/files/systemd/magma_dp_envoy.service

```diff
@@ -19,7 +19,7 @@ Type=simple
 EnvironmentFile=/etc/environment
 # Add delay to let envoy-controller init
 ExecStartPre=/bin/sleep 40
-ExecStart=/sbin/ip netns exec envoy_ns1 /usr/bin/envoy --bootstrap-version 2 -c /var/opt/magma/envoy.yaml --log-path /var/log/envoy.log
+ExecStart=/sbin/ip netns exec envoy_ns1 /usr/bin/envoy --bootstrap-version 3 -c /var/opt/magma/envoy.yaml --log-path /var/log/envoy.log
```

Comment:
Given this https://github.com/envoyproxy/envoy/pull/17724, perhaps we can remove this `--bootstrap-version` flag entirely?

Reply:
Let's remove it then.